### PR TITLE
Upgrade Ubuntu 20.04

### PR DESCRIPTION
**Pull request by adenishchenko**

### CODE REVIEW

**Comment by WadeBarnes** [comment]
(ci) test this please

**Comment by WadeBarnes** [comment]
@adenishchenko, What is the relation of this PR to the one @ryMarsh44 started here; [WIP: Ubuntu 20.04 upgrade](https://github.com/hyperledger/indy-node/pull/1652)?

**Comment by WadeBarnes** [comment]
@adenishchenko, Please edit the PR and target the `ubuntu-20.04-upgrade` branch as the destination.

**Comment by WadeBarnes** [comment]
@adenishchenko, Also if this PR is still a work in progress, please edit the PR and change it to a Draft PR until it is ready for final review.

**Comment by WadeBarnes** [comment]
@ryMarsh44, @Toktar, @adenishchenko, What is the relationship between this PR and https://github.com/hyperledger/indy-node/pull/1652?  Can https://github.com/hyperledger/indy-node/pull/1652 be closed in favor of this PR?

**Comment by WadeBarnes** [comment]
@ryMarsh44, @Toktar, @adenishchenko, @udosson, If I understand correctly, based on the Indy Contributors call today, this PR can be taken out of **WIP** status.  Is that correct?

**Comment by WadeBarnes** [comment]
DCO Sign-off is missing from this commit; https://github.com/hyperledger/indy-node/pull/1667/commits/9959164eb02b0aac43110055a8e3f43c5e30587a

**Review by WadeBarnes — CHANGES_REQUESTED** [review]
See comments

**Comment by askolesov** [comment]
@WadeBarnes
* PR duplicates: this is the main PR with upgrade work, #1652 can be closed I think.
* WIP status: yes, it can be taken out of it.
* CI: this PR doesn't rely on Jenkins because Jenkins is only for 16.04. We definitely need to merge CI/CD work for Ubuntu 20.04 here, but I don't see a PR. (Sorry for my confusing PR, I've just closed it)
* Signature: I think it's better to fix it right before merge.

**Comment by WadeBarnes** [comment]
> @WadeBarnes
> * Signature: I think it's better to fix it right before merge.

Yes, this needs to be fixed sooner than later.

**Comment by askolesov** [comment]
Seems that it's ready now. @WadeBarnes Could you review it one more time, please?

**Comment by adenishchenko** [comment]
Hello @WadeBarnes . Could you review this PR?

File: dev-setup/ubuntu/ubuntu-2004/SetupVMTest.txt
Comment (WadeBarnes) [inline dev-setup/ubuntu/ubuntu-2004/SetupVMTest.txt:38]: This needs to be upgraded.

**Review by WadeBarnes — CHANGES_REQUESTED** [review]
One item that needs to be addressed.

File: dev-setup/ubuntu/ubuntu-2004/SetupVMTest.txt
Comment (Toktar) [reply dev-setup/ubuntu/ubuntu-2004/SetupVMTest.txt:38]: @WadeBarnes I totally agree with you, we need to update Pygments version. But Indy Node does not have a pinned dependency on Pygments package. And this file is just a set of recommendations for a virtual machine, relevant at the moment.
Thus, I propose not to block this PR and merge it, since Indy Node has nothing to do with Pygments package. Only after updating this package in Plenum does it make sense to update or delete this doc.

**Review by WadeBarnes — APPROVED** [review]

**Comment by WadeBarnes** [comment]
@Toktar, It looks like the code owner's file has not been updated on this branch yet.  So you'll have to approve and merge.

**Review by Toktar — APPROVED** [review]

**Pull request merged**
